IntegrationTests/CarthageIntegration/purchases-root

```diff
@@ -1 +0,0 @@
-../../
```

Comment:
do we need this file or it can be removed?

Reply:
I needed to update that. Changing back to what it was.